src/coreclr/jit/codegenarm64.cpp

```diff
@@ -203,7 +203,14 @@ void CodeGen::genPopCalleeSavedRegistersAndFreeLclFrame(bool jmpEpilog)
     }
 
     JITDUMP("    calleeSaveSpOffset=%d, calleeSaveSpDelta=%d\n", calleeSaveSpOffset, calleeSaveSpDelta);
-    genRestoreCalleeSavedRegistersHelp(regsToRestoreMask, calleeSaveSpOffset, calleeSaveSpDelta);
```

Comment:
It is possible to adjust the SP only here.
So we can simplify the `genRestoreCalleeSavedRegistersHelp` and this is advantageous for all arches.